applications/ecallmgr/src/ecallmgr_originate.erl

```diff
@@ -768,6 +768,8 @@ maybe_send_originate_uuid({'fs', UUID}, Pid, #state{server_id=ServerId
                                                    }) ->
     CtlQ = gen_listener:queue_name(Pid),
     publish_originate_uuid(ServerId, UUID, JObj, CtlQ);
+maybe_send_originate_uuid({'api', UUID}, Pid, State) ->
+    maybe_send_originate_uuid({'fs', UUID}, Pid, State);
```

Comment:
@lazedo yes, however the control queue for the UUID is not - it is delivered by the originate_uuid event after an ecallmgr_call_control proc is spawned/initialized